aws-predictions/src/main/java/com/amplifyframework/predictions/aws/adapter/IdentifyResultTransformers.java

```diff
@@ -108,4 +124,50 @@ public static Pose fromRekognitionPose(@NonNull com.amazonaws.services.rekogniti
 
         return amplifyLandmarks;
     }
+
+    /**
+     * Gets all the binary features from AWS Rekognition's face
+     * details and compiles them into a single list.
+     * @param face the Rekognition face detail object
+     * @return the list of Amplify {@link BinaryFeature}
+     */
+    public static List<BinaryFeature> fromFaceDetail(FaceDetail face) {
+        return Arrays.asList(
+                BinaryFeature.builder()
+                        .type("Beard")
```

Comment:
Does Rekognition have labeled constants somewhere for like `"Beard"` ,  etc., that you could use instead of hard-coding the string?

Reply:
they are individual features with strongly typed classes in Rekognition, but iOS folks decided to make them all into generic binary attributes (binary features). So `FaceDetail#getBeard()` returns a `Beard` typed object, which makes it hard to obtain constants. Perhaps I can do something like `Beard.class.getSimpleName()`, but that seems somewhat pointless.

Reply:
^The above solution would also cause SLIGHT disparity with iOS since the binary feature for glasses is hardcoded as "EyeGlasses", but the Rekognition class is named "Eyeglasses"

Reply:
Hmm, perhaps so. These are specific types of binary features that OUR implementation happens to detect, so that wouldn't belong in the core. However, I can see it being useful inside the actual plugin module.

I'll make a new PR to address this since it is big enough of a change to add to an already approved PR.